spec/openapi.yaml

```diff
@@ -3453,6 +3325,125 @@ paths:
         500:
           $ref: '#/components/responses/Error500ResponseDefinition'
 
+  /context-api/v2/projects/{projectId}/bindings/list:
+    post:
+      summary: Get bindings
+      description: |-
+        Return bindings for specified parameters.
```

Comment:
`Returns a paginated list of bindings according to specified parameters.`

Reply:
There is no info about page size

Reply:
fixed